packages/field-highlighter/src/vaadin-user-tags.js

```diff
@@ -132,8 +162,35 @@ export class UserTags extends PolymerElement {
   }
 
   /** @private */
-  __targetChanged(target) {
-    this.$.overlay.positionTarget = target;
+  __onTargetVisibilityChange(isVisible) {
+    this.__isTargetVisible = isVisible;
+
+    if (isVisible && this.__flashQueue.length > 0 && !this.flashing) {
```

Comment:
**note:** The case where the field enters the screen while the flashing animation is already running seems to be quite rare. However, I decided to add a check for the sake of safety. Any opinions?

Reply:
Added a unit test.